styles/components/_processSection.scss

```
@@ -0,0 +1,116 @@
+.processSection {
+  &__title {
+    text-align: center;
+    font-size: 26px;
+    font-weight: 400;
+    line-height: 30px;
+    max-width: 60vw;
+    margin: 0 auto 24px auto;
+  }
+
+  &__items {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+  }
+
+  &__item {
+    $item: &;
+
+    list-style: none;
+
```

```suggestion
    margin-top: 24px;
```

Para aplicar el valor que hay en figma